manim/mobject/types/image_mobject.py

```diff
@@ -27,8 +27,8 @@ class AbstractImageMobject(Mobject):
     ----------
     scale_to_resolution : :class:`int`
         At this resolution the image is placed pixel by pixel onto the screen, so it will look the sharpest and best.
-        This is a custom parameter of ImageMobject so that rendering a scene with the `--quality low` flag for faster rendering and testing won't effect the position of the image on the screen.
-        Calculated by `height = image_height / scale_to_resolution * config["frame_height"]`
+        This is a custom parameter of ImageMobject so that rendering a scene with the ``--quality low`` flag for faster rendering and testing won't effect the position of the image on the screen.
+        Calculated by ``height = image_height / scale_to_resolution * config["frame_height"]``
```

Comment:
Nitpick: I believe that this is too detailed for a parameter description. Implementation specific details can be inferred from the code, if someone is interested.

Reply:
I will delete this then.